instrumentation/elasticsearch/lib/opentelemetry/instrumentation/elasticsearch/instrumentation.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Elasticsearch
+      # The Instrumentation class contains logic to detect and install the Elasticsearch instrumentation
+      class Instrumentation < OpenTelemetry::Instrumentation::Base
+        install do |_config|
+          require_dependencies
+          patch_client
+        end
+
+        present do
+          defined?(::Elasticsearch::Transport)
+        end
```

Comment:
@mmontalvo - is there any minimum version that we need to check for? Or does this work for all versions?

(An example of how to do a version check is [here](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/instrumentation/pg/lib/opentelemetry/instrumentation/pg/instrumentation.rb#L23-L25))

Reply:
From what I found, `perform_request` has been set to current implementation from [this commit](https://github.com/elastic/elasticsearch-ruby/commit/b7b1835638f69c30b9c1fb890a84d4e36806e3d8#diff-815d56877734c3f8654aeb742d159115bb48be9f6f775570a53d3e69a5be8f60R128)
I think we shouldn't need to worry about minor versions.

Reply:
I think they [introduced the method](https://github.com/elastic/elasticsearch-ruby/commit/a2bbb9c5b6b3cb0f062d751eca975843502b571b) on May 17, 2013, which would put the minimum supported version around `0.1.1` - so I agreee, we probably don't need a minimum version check here. 😆